src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -52,8 +55,25 @@
  */
 public class ManualCondition extends PromotionCondition {
     private String users;
+        
     private List<ParameterDefinition> parameterDefinitions = new ArrayList<ParameterDefinition>();
     
+    //TODO ======================
+    //to use Custom label for approve button, add deamon property 'MANUAL_CONDITION_APPROVE_BUTTON_CUSTOM_LABEL' to "/etc/init.d/jenkins" file in value to JAVA_CMD
+	//sample: JAVA_CMD="$JENKINS_JAVA_CMD $JENKINS_JAVA_OPTIONS -DJENKINS_HOME=$JENKINS_HOME -DMANUAL_CONDITION_APPROVE_BUTTON_CUSTOM_LABEL=CUSTOM_VALUE -jar $JENKINS_WAR"
+    
+    @Nonnull
+    private String executeButtonLabel;
+    
+    public String getExecuteButtonLabel() {
+    	String label = System.getProperty("MANUAL_CONDITION_APPROVE_BUTTON_CUSTOM_LABEL");
+    	if(null == label) {
+    		label = "Approve";
```

Comment:
Regarding naming of the property, can the property be named as 'ManualCondition_ApproveButtonCustomLabel" ?

Regarding Localization of Strings, can you point me the place where you've used localization in your code, so that I can make the changes accordingly.